src/utils/reversal.ts

```diff
@@ -0,0 +1,45 @@
+import { models, MessageRecord, ChatRecord, ContactRecord } from '../models'
+import * as socket from './socket'
+import constants from '../constants'
+import * as jsonUtils from './json'
+
+interface ReversalInput {
+  tenant: number
+  type: string
+  errorMsg: string
+  msgUuid: string
+  chat: ChatRecord
+  sender: ContactRecord
+}
+
+export async function reversal({
```

Comment:
I'm confused do why don't we send back the sats here and only in /network/send.ts

Reply:
So sorry, the function name is misleading, I changed the name in my latest push